src/components/MasterItemsSelect.js

```diff
@@ -6,8 +6,10 @@ class MasterItemsSelect extends Component {
     super(props);
     this.handleChange = this.handleChange.bind(this);
     this.handleChangeCategory = this.handleChangeCategory.bind(this);
+    const filteredItems = this.filterItems(this.props.masterItems[0].category);
     this.state = {
-      filteredItems: this.filterItems(this.props.masterItems[0].category)
+      filteredItems,
+      selectedItemId: filteredItems[0].id
```

Comment:
Ah I see what you mean.. yeah I feel like there must be a better way, too.. I can take a closer look at it. But this seems okay for now. Will merge.